src/python/sortingview/backend/task_function_ids.json

```diff
@@ -7,5 +7,6 @@
     "sortingview_workspace_action.1", "sortingview_workspace_sorting_curation_action.1",
     "fetch_unit_metrics.1", "fetch_spike_amplitudes.1",
     "latency_test_query.1", "get_action_latency_test_subfeed.1", "subfeed_latency_test_append.1",
-    "sortingview_get_python_package_version.1"
+    "sortingview_get_python_package_version.1",
+    "get_best_matching_units.2"
```

Comment:
new backend task function called get_best_matching_units. It compares the sorting and for each unit in sorting A it finds the best matching unit in sorting B.

Reply:
These relationships are not necessarily unique or symmetrical, right? We could have 1A and 2A whose best match is both 3B, etc?

Reply:
good question. This depends on the type of matching algorithm on the python side. Some may be one-to-one and some may be one-to-many.